Automattic/wp-calypso
# Jetpack Connect: add WP logo in the Jetpack remote install credentials form.

Main change introduced here is the WP logo added to `/public/images/wordpress` and included in the credentials form in the remote install flow.

Styling extras:
- use Card component to introduce extra spacing between the form and the footer,
- center the header text.

**To test:**
- checkout this branch on your local Calypso,
- go to `calypso.localhost:3000/jetpack/connect` and verify the visual changes.

**Note:** PLEASE IGNORE password hide icon inside the input field and button color. It is changed in a follow-up PR.

![screen shot 2018-02-28 at 22 33 30](https://user-images.githubusercontent.com/13561163/36817381-dd4b9742-1cd8-11e8-9c35-712d55f22b51.png)
